### DISCUSSION TITLE
8.0.1 Released

### DISCUSSION THREAD

**Discussion by adrianhall** [CommunityToolkit/Datasync#105] [Announcements]
## What's Changed
* Fixing minor typos by @Keboo in https://github.com/CommunityToolkit/Datasync/pull/86
* Removed obsolete Microsoft.AspNetCore.Http.Abstractions by @Keboo in https://github.com/CommunityToolkit/Datasync/pull/85
* Leverage CallerArgumentExpression by @Keboo in https://github.com/CommunityToolkit/Datasync/pull/87
* (#90) Fixed threading issue in PushAsync by @adrianhall in https://github.com/CommunityToolkit/Datasync/pull/91
* (#96) authentication provider. by @adrianhall in https://github.com/CommunityToolkit/Datasync/pull/97
* (#99) Make service client public. by @adrianhall in https://github.com/CommunityToolkit/Datasync/pull/104

## New Contributors
* @Keboo made their first contribution in https://github.com/CommunityToolkit/Datasync/pull/86

**Full Changelog**: https://github.com/CommunityToolkit/Datasync/compare/8.0.0...8.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/CommunityToolkit/Datasync/releases/tag/8.0.1'>8.0.1</a>.</em>

**Comment by adrianhall** [comment]
… and, in case anyone is waiting for 8.0.2 - unless there are actionable issues filed between now and Friday, I’ll do the release a week early on October 4th.

**Comment by Shahid-khan5** [comment]
Could you include a migration guide in the documentation for transitioning from Azure Mobile Apps to Data Sync?

**Reply by adrianhall** [reply]
From the server side, it's basically "change to the new nuget packages and recompile"

From the client side, it's too complex since it requires a completely different setup.   Given it requires a rewrite to EF Core, I'm reluctant to spend the time on it.  I'll happily accept a PR to the docs if you want to write one though.

**Reply by Shahid-khan5** [reply]
Got it. I'll try to write the docs for migration.
